understand-conten-marketing/readme.md

```diff
@@ -0,0 +1,76 @@
+> Goal of this guide: to help you understand what content marketing is.
+
+# What the flying flop is content marketing?
+
+In *The Rise of Storytelling as the New Marketing*, Joe Pulizzi describes content marketing as “the idea that storytelling is key to attract[ing] and retain[ing] customers.”
+
+The first known example of content marketing was the creation of a magazine, first published in 1895 by John Deere (a company that sells tools for farmers), called *the Furrow*. It’s goal was not to sell farm equipment, but to provide knowledge on farming, making John Deere an expert on farming.
+
+By doing this, John Deere gave valuable information to their readers. The value that the readers received from the magazine was directly linked to the sender, John Deere. The value the reader now linked to John Deere greatly impacted how they valued all other things John Deere did, including selling farm equipment.
+
+And sure enough, when those farmers needed new gear they would go to the expert they knew and trusted: John Deere.
+
+The next few quotes are from the *Journal of Marketing Management* in the article “Understanding Digital Content Marketing”, in which Jenifer Rowley describes some key notions about content marketing:
+
```

Comment:
I understand that the achievable is to science this, but these quotes are really difficult to understand and don't really add value because a) they are too complex for the average reader, or even a good reader, b) some of them are incomplete thoughts, and c) there are no conclusions drawn from them until after the next header, which is confusing.